.github/workflows/build-with-bal-test-native.yml

```diff
@@ -2,76 +2,16 @@ name: GraalVM Check
 
 on:
   schedule:
-    - cron: '30 18 * * *'
+    - cron: "30 18 * * *"
   workflow_dispatch:
 
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    services:
-      mongodb:
-        image: mongo:4.2.0
-        env:
-          # Following credentials are only used during testing in docker container
-          MONGO_INITDB_ROOT_USERNAME: admin
-          MONGO_INITDB_ROOT_PASSWORD: admin
-        ports: 
-          - 27011:27017
-      mongodb-ssl:
-        image: rzhilkibaev/mongo-x509-auth-ssl:latest
-        ports: 
-          - 27012:27017
-
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          distribution: 'temurin'
-          java-version: 17.0.7
-
-      - name: Set Up Ballerina
-        uses: ballerina-platform/setup-ballerina@v1.1.0
-        with:
-          version: latest
-
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
 
-      - name: Build with Gradle
-        env:
-          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
-          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
-          JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
-        run: |
-          ./gradlew build -x test
-
-      - name: Set up GraalVM
-        uses: graalvm/setup-graalvm@v1
-        with:
-          java-version: '17'
-          distribution: 'graalvm-community'
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          set-java-home: 'false'
-
-      - name: Check GraalVM installation
-        run: |
-          echo "GRAALVM_HOME: ${{ env.GRAALVM_HOME }}"
-          echo "JAVA_HOME: ${{ env.JAVA_HOME }}"
-          native-image --version
-
-      - name: Run Ballerina tests(MongoDB-SSL) using the native executable
-        run: bal test --graalvm --groups mongodb-ssl ./ballerina
-        env:
-          JAVA_HOME: /usr/lib/jvm/default-jvm
-          # Following credentials are only used during testing in docker container
-          MONGODB_USER: C=US,ST=CA,L=San Francisco,O=Jaspersoft,OU=JSDev,CN=admin
-
-      - name: Run Ballerina tests(MongoDB) using the native executable
-        run: bal test --graalvm --groups mongodb ./ballerina
-        env:
-          JAVA_HOME: /usr/lib/jvm/default-jvm
-          # Following credentials are only used during testing in docker container
-          MONGODB_USER: admin
-          MONGODB_PASSWORD: admin
+jobs:
+  call_stdlib_workflow:
+    name: Run StdLib Workflow
+    if: ${{ github.event_name != 'schedule' || (github.event_name == 'schedule' && github.repository_owner == 'ballerina-platform') }}
```

Comment:
Can we omit using the word `stdlib` in connector repos, if possible?

Reply:
+1. Fixed with 91aca74ff7e8b9c62e5381e266cffb5b1f63e68b